include/irr/asset/filters/CSwizzleableAndDitherableFilterBase.h

```diff
@@ -17,20 +17,124 @@ namespace irr
 	{
 		namespace impl
 		{
+			namespace detail
+			{
+				union NormalizeValues
```

Comment:
you want a `struct` and then an "anonymous union" of the member variables

Reply:
not an issue, discussed on the dc